### ISSUE TITLE
Console log UI takes approx 10 secs to load for jobs with console log of 40-50K lines

### ISSUE THREAD

**Issue by rajiesh** [gocd/gocd#3371]
##### Issue Type

- Bug Report

###### Basic environment details

* Go Version: `17.4.0 (4823-6c8c66c4085fce499389c817732bedf570bc4c97)`

##### Steps to Reproduce

With Console log UI improvements(https://github.com/gocd/gocd/pull/3199) the console log tab takes ~10 sec to load for jobs with log of 40-50K lines

From a user perspective its a bit confusing when they see the log tab without any contents for ~10 secs, since there are no visual representation to say its loading the logs. Can the load time be reduced for huge logs? or having a spinner as visual representation for loading a good idea in case of delay?

`Installer-tests` and `UploadInstallers` pipelines in build.go.cd are places were we can see this issue

**Comment by fire** [comment]
Maybe it's possible to fold the console logs? https://blog.travis-ci.com/2013-05-22-improving-build-visibility-log-folds/

**Comment by rajiesh** [comment]
Verified on 17.4.0 (4888-cbd790d635c52dc6e0765cc2b4c8e52745e00271). The Console log loading message gives users indication of the loading. However the page loading still takes time.

Also the animation for `Waiting for console logs..` message increases the page CPU utilization, during test when we profiled on the Chrome browser for the tab which opens the console log page, we could observe ~30% CPU utilization by that page.

We will close this issue for now, if users feel the waiting and the CPU utilization annoying will re-open the issue.
